### ISSUE TITLE
Software Launch Failure

### ISSUE THREAD

**Issue by VijoySimon** [danielbuechele/atemOSC#151]
Hi there, I am currently running Mac 10.11.6 (El Captain) and Atem Software control v7.5 . When i try running ATEMOSC 3.0.5, i get this error (see attached screenshot). 

Can i use your program without having to downgrade Atem Software control or having to Update my Mac OS. If I needed to upgrade the mac OS, would it work on Mohave (10.14)?

Let me know! 

![Screen Shot 2020-04-23 at 11 42 53 AM](https://user-images.githubusercontent.com/64218600/80127032-5f8ed080-8559-11ea-95cf-f5cd819490c5.png)

**Comment by SteffeyDev** [comment]
You will need to either upgrade your switcher to 8.1, or run an older version of atemOSC. In the list of releases, you may be able to find the version that supported the 7.5 SDK.

**Comment by SteffeyDev** [comment]
Actually, not sure if we ever supported 7.5, we may have jumped from 7.4 to 8.1. You can try various older versions to see if any of them work with 7.5, though. We only officially support the latest version 8.1.

**Comment by sneat** [comment]
@SteffeyDev I wonder if we should investigate being able to detect the version and load the appropriate resources to allow us to support older version switcher installs, without forcing using an older version of atemOSC?

**Comment by VijoySimon** [comment]
Hi Blair,

After I sent the original request I saw your emails but I was busy working on another project.

Tomorrow when I’m back at work, I will try and see if I can update the Black Magic Design ATEM Studio HD that we have in a room to version 8.1.

If I don’t have to update the Mac OS yet and I don’t have to uninstall atem and reinstall (thus potentially losing all configuration settings) I would consider that a win!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aBigRobot** [comment]
Hey there,

Very interested to start using this app, but I am having the same issue above. I am running ATEM 7.3, and I am on MacOS 10.13.4. Any chance that this could be easily updated to support the older ATEM software? I can't update to 8.1 without updating OS and I am trying to avoid doing that on an older machine. 

I was able to get an older version of ATEMosc to connect and function correctly, but it did not have the ability to use macros which I am really hoping to use!

Thanks for your help, this app is AWESOME and necessary!

**Comment by SteffeyDev** [comment]
@sneat I'll open that as a different issue, don't want to confuse this one.

@VijoySimon You can save your ATEM switcher configuration as XML, and then you can be sure you won't lose it when you upgrade.

**Comment by VijoySimon** [comment]
Hey everyone,

I was able to get it all to work without having to upgrade switcher or mac os. I used version 2.5.6 of AtemOSC.

Thanks!

When we do eventually get a new mac mini and have mojave, I’m excited to get all added features but for now just  having a super large touch button on an ipad to make auto transitions is fantastic!

Vijoy Simon
Media Technician | Church of the Rock
1397 Buffalo Place, Winnipeg MB, R3T 1L6
p: (204) 261-0070 ~ ext: 275| f: (204) 261-0066 e: vijoy@churchoftherock.ca<mailto:vijoy@churchoftherock.ca>
KNOW GOD  LIVE FREE  FIND PURPOSE
